docs/specs/git.yml

```diff
@@ -58,19 +58,16 @@ outputs:
   build.manifest:
 ---
 # Specify invalid author in YAML header should log warning
-# The warning message for author-not-found can one of the following:
-# - Author 'invalid_user' cannot be recognized: User not found on GitHub
-# - Author 'invalid_user' cannot be recognized: GitHub API rate limit exceeded
 repo: https://github.com/docascode/docfx-test-dependencies#git-test-3
 outputs:
   README.json:
   testdata1.json:
   build.manifest:
   dep.json:
   build.log: |
-    ["warning","author-not-found","*","README.md"]
+    ["warning","*","*","README.md"]
```

Comment:
can this error code and message be deterministic?

Reply:
This can be either `author-not-found` or `resolve-author-failed` depending on the rate limit status